src/sentry/api/urls.py

```diff
@@ -548,6 +554,16 @@
                     OrganizationAlertRuleIndexEndpoint.as_view(),
                     name="sentry-api-0-organization-alert-rules",
                 ),
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/alert-rules/(?P<alert_rule_id>[^\/]+)/triggers/(?P<alert_rule_trigger_id>[^\/]+)$",
+                    OrganizationAlertRuleTriggerDetailsEndpoint.as_view(),
+                    name="sentry-api-0-organization-alert-rule-trigger-details",
+                ),
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/alert-rules/(?P<alert_rule_id>[^\/]+)/triggers$",
```

Comment:
Should this have a trailing slash like our other APIs?

Reply:
Yep, will fix